src/app/pages/subscriptions/manage-subscription.jsx

```diff
@@ -130,113 +202,165 @@ function ChplManageSubscription(props) {
-      </Typography>
-      <Typography>
-        { subscriber.email }
-        {' | '}
-        { subscriber.role.name }
-      </Typography>
-      <Typography>
-        { subscriptions.reduce((sum, subscription) => sum + subscription.subscriptions.length, 0) }
-        {' '}
-        Total Subscriptions
-      </Typography>
-      <Button
-        disabled={selectedSubscriptions.length === 0}
-        className={classes.deleteButton}
-        endIcon={<DeleteIcon />}
-      >
-        Remove Selected (
-        { selectedSubscriptions.length }
-        )
-      </Button>
-      { subscriptions.map((subscription) => (
-        <Accordion
-          className={classes.criterionAccordion}
-          onChange={() => handleAccordionChange(subscription)}
-          id={`subscription-${subscription.certifiedProductId}`}
-          key={`subscription-${subscription.certifiedProductId}`}
-        >
-          <AccordionSummary
-            className={classes.criterionAccordionSummary}
-            expandIcon={getIcon(subscription)}
-            id={`subscription-id-${subscription.certifiedProductId}-header`}
-          >
-            <Box className={classes.criterionAccordionSummaryHeader}>
-              <Box className={classes.criterionAccordionSummarySubBox}>
-                <Box className={classes.criterionAccordionSummaryData}>
-                  <Typography>
-                    {subscription.chplProductNumber}
-                  </Typography>
-                </Box>
-                <Box className={classes.criterionAccordionSummaryData}>
-                  <Typography variant="subtitle1">
-                    { subscription.subscriptions.length }
-                    {' '}
-                    subscription
-                    { subscription.subscriptions.length !== 1 && 's' }
-                  </Typography>
-                </Box>
-                <Box className={classes.criterionAccordionSummaryData}>
-                  checkbox
-                </Box>
-              </Box>
+    <Box className={classes.page}>
+      <Box className={classes.header}>
+        <Container className={classes.headerContent}>
+          <Typography variant="h1">
+            My Subscriptions
+          </Typography>
+          <Typography>
+            {subscriber.email}
+            {' | '}
+            {subscriber.role.name}
+          </Typography>
+        </Container>
+      </Box>
+      <Container>
+        <Box className={classes.chplLeftHandFormat}>
+          <Card>
+            <Box className={classes.mySubsciptionImagery} />
+            <CardContent>
+              <Typography gutterBottom variant="h4" component="h2"><strong>Welcome to your subscription page!</strong></Typography>
+              <Typography>Here, you can easily view and manage your subscriptions.</Typography>
+            </CardContent>
+          </Card>
+          <Box>
+            <Box display="flex" flexDirection="row" justifyContent="space-between" pb={4} alignItems="center">
```

Comment:
Any particular reason these attributes aren't in a style? They're all (almost) css attributes, right?

Reply:
Yes, let me fix this